File: main/java/com/escalones/control12/modelos/DatosEmpresa.java
package com.escalones.control12.modelos;

/**
 * Clase DatosEmpresa
 * Representa los datos básicos de una empresa.
 * 
 * @author xarxa
 */
public class DatosEmpresa {

    /**
     * Identificador único de la empresa.
     */
    private Integer id;

    /**
     * Nombre de la empresa.
     */
    private String nombre;

    /**
     * CIF (Código de Identificación Fiscal) de la empresa.
     */
    private String cif;

    /**
     * Dirección física de la empresa.
     */
    private String direccion;

    /**
     * Código postal de la empresa.
     */
    private String cp;

    /**
     * Ciudad donde se ubica la empresa.
     */
    private String ciudad;

    /**
     * Primer número de teléfono de la empresa.
     */
    private String telefono1;

    /**
     * Segundo número de teléfono de la empresa.
     */
    private String telefono2;

    /**
     * Correo electrónico de la empresa.
     */
    private String email;

    /**
     * Sitio web de la empresa.
     */
    private String web;

    /**
     * Constructor vacío.
     */
    public DatosEmpresa() {}

    // Getters y Setters
    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getCif() {
        return cif;
    }

    public void setCif(String cif) {
        this.cif = cif;
    }

    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    public String getCp() {
        return cp;
    }

    public void setCp(String cp) {
        this.cp = cp;
    }

    public String getCiudad() {
        return ciudad;
    }

    public void setCiudad(String ciudad) {
        this.ciudad = ciudad;
    }

    public String getTelefono1() {
        return telefono1;
    }

    public void setTelefono1(String telefono1) {
        this.telefono1 = telefono1;
    }

    public String getTelefono2() {
        return telefono2;
    }

    public void setTelefono2(String telefono2) {
        this.telefono2 = telefono2;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getWeb() {
        return web;
    }

    public void setWeb(String web) {
        this.web = web;
    }
}

File: src/Entity/DatosEmpresa.php
<?php

namespace App\Entity;

use App\Repository\DatosEmpresaRepository;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: DatosEmpresaRepository::class)]
class DatosEmpresa
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id = null;

    #[ORM\Column(length: 150)]
    private ?string $nombre = null;

    #[ORM\Column(length: 20)]
    private ?string $cif = null;

    #[ORM\Column(length: 255)]
    private ?string $direccion = null;

    #[ORM\Column(length: 10)]
    private ?string $cp = null;

    #[ORM\Column(length: 100)]
    private ?string $ciudad = null;

    #[ORM\Column(length: 15)]
    private ?string $telefono1 = null;

    #[ORM\Column(length: 15)]
    private ?string $telefono2 = null;

    #[ORM\Column(length: 100)]
    private ?string $email = null;

    #[ORM\Column(length: 100)]
    private ?string $web = null;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNombre(): ?string
    {
        return $this->nombre;
    }

    public function setNombre(string $nombre): static
    {
        $this->nombre = $nombre;

        return $this;
    }

    public function getCif(): ?string
    {
        return $this->cif;
    }

    public function setCif(string $cif): static
    {
        $this->cif = $cif;

        return $this;
    }

    public function getDireccion(): ?string
    {
        return $this->direccion;
    }

    public function setDireccion(string $direccion): static
    {
        $this->direccion = $direccion;

        return $this;
    }

    public function getCp(): ?string
    {
        return $this->cp;
    }

    public function setCp(string $cp): static
    {
        $this->cp = $cp;

        return $this;
    }

    public function getCiudad(): ?string
    {
        return $this->ciudad;
    }

    public function setCiudad(string $ciudad): static
    {
        $this->ciudad = $ciudad;

        return $this;
    }

    public function getTelefono1(): ?string
    {
        return $this->telefono1;
    }

    public function setTelefono1(string $telefono1): static
    {
        $this->telefono1 = $telefono1;

        return $this;
    }

    public function getTelefono2(): ?string
    {
        return $this->telefono2;
    }

    public function setTelefono2(string $telefono2): static
    {
        $this->telefono2 = $telefono2;

        return $this;
    }

    public function getEmail(): ?string
    {
        return $this->email;
    }

    public function setEmail(string $email): static
    {
        $this->email = $email;

        return $this;
    }

    public function getWeb(): ?string
    {
        return $this->web;
    }

    public function setWeb(string $web): static
    {
        $this->web = $web;

        return $this;
    }
}
